### PR TITLE
feat: added articles/news component

### PR DESCRIPTION
**Pull request by johnmarkovalo**
Re: #49 

This PR includes:
- added articles/news component but with static data and need to fix the margin. See photo below.

image

### CODE REVIEW

File: components/ArticleCard.vue [hunk 1]
```diff
@@ -0,0 +1,33 @@
+<template>
+  <div class="rounded-lg bg-white mb-8 py-6">
+    <h3 class="text-primary text-center font-semibold text-sm lg:text-base">Latest News</h3>
+    <div v-if="articles">
+      <div v-for="article in articles" :key="article.id" class="mx-10 my-5">
+        <h4  class="text-purple-900 font-semibold text-5sm lg:text-7sm">{{article.title}}</h4>
+        <h4  class="text-primary font-semibold text-sm lg:text-6sm">{{article.sourceName}}</h4>
```
Comment (jbienesdev) [inline components/ArticleCard.vue:7]: There's no `text-7sm` or `text-6sm` class

Comment (johnmarkovalo) [reply components/ArticleCard.vue:7]: noted. Will fix it later

**Comment by jbienesdev** [comment]
@johnmarkovalo I will leave this PR on queue until we are finished with migrating everything from firebase and news feed is already in place in Strapi.

**Comment by jbienesdev** [comment]
@JHNLWHD fixed merge conflict of this issue. Here's the output

covid11

Thanks @johnmarkovalo 👍

File: components/ArticleCard.vue [hunk 2]
```diff
@@ -0,0 +1,33 @@
+<template>
+  <div class="rounded-lg bg-white mb-8 py-6">
+    <h3 class="text-primary text-center font-semibold text-sm lg:text-base">Latest News</h3>
+    <div v-if="articles">
+      <div v-for="article in articles" :key="article.id" class="mx-10 my-5">
+        <h4  class="text-purple-900 font-semibold text-base">{{article.title}}</h4>
+        <h4  class="text-primary font-semibold text-sm ">{{article.sourceName}}</h4>
```
Comment (JHNLWHD) [inline components/ArticleCard.vue:7]: Add a hyperlink behavior which is to open link in new tab.

Comment (jbienesdev) [reply components/ArticleCard.vue:7]: Will add on my next PR when linking news to strapi. Just retained the code on this PR.

Comment (JHNLWHD) [reply components/ArticleCard.vue:7]: Okay @jbienesdev . 👍

**Comment by JHNLWHD** [comment]
@jbienesdev please fix the conflicts

**Comment by jbienesdev** [comment]
@JHNLWHD squash and merge this PR.

**Pull request merged**